CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 3.0.0
+
+## Package Updates
```

Comment:
Will need some instructions here on what changes a developer needs to perform in their app for a successful upgrade. You may learn more about what these steps are from testing it in GAC.